common/components/componentsBySection/FindProjects/Filters/RenderFilterCategory.jsx

```diff
@@ -34,11 +34,14 @@ class RenderFilterCategory<T> extends React.PureComponent<Props<T>, State> {
     let cs = _.concat(c, s);
     //cs is now an array of each key we want to use for expand/collapse tracking
 
-    //make an object and push in each key from cs, set all values to false
+    //make an object and push in each key from cs, set all values based wether the tag inside is checked
```

Comment:
```suggestion
    //make an object and push in each key from cs, set all values based whether the tag inside is checked
```

Reply:
The typo reappeared.  If you make a text commit via github, make sure you merge the commit into your code branch before making further changes.

Reply:
just sent the adjust on the comment, sorry about that